implementations/rust/ockam/ockam_command/src/project/ticket.rs

```diff
@@ -76,8 +80,12 @@ pub struct TicketCommand {
     usage_count: Option<u64>,
 
     /// Name of the relay that the identity using the ticket will be allowed to create. This name is transformed into attributes to prevent collisions when creating relay names. For example: `--relay foo` is shorthand for `--attribute ockam-relay=foo`
-    #[arg(long = "relay", value_name = "ENROLEE_ALLOWED_RELAY_NAME")]
+    #[arg(long = "relay", value_name = "ENROLLEE_ALLOWED_RELAY_NAME")]
     allowed_relay_name: Option<String>,
+
+    /// Add the enroller role to your ticket. If you specify it, this flag is transformed into the attributes `--attribute ockam-role=enroller`
```

Comment:
In the long help text it would be good to specify what is the effect of setting this attribute. Why would anyone want to do this?

Reply:
great idea. i add the "why" content.